docs/pages/dev/reference/API/org_unit_change_request_configuration.md

```diff
@@ -0,0 +1,240 @@
+# `OrgUnitChangeRequestConfiguration` API
+
+This document is a first draft of the `OrgUnitChangeRequest` API that was designed by Benjamin in August 2024 and updated by Thibault.
+It still needs to be updated and completed, once the developments are over.
+ 
+## 1 - POST /api/change_requests_configs/
+
+API used to create or modify a Change Request configuration.
+
+### Permissions
+
+- Change Request Configuration
+
+### Body
+
+```json
+{
+  "project_id": "Int - Project ID",
+  "org_unit_type_id": "Int - OrgUnit Type ID",
+  "org_units_editable": "Boolean - Whether or not OrgUnits of this OrgUnit Type are editable",
+  "editable_fields": ["Array<String> - List of possible fields"],
+  "possible_parent_types": ["Array<Int> - List of possible OrgUnit Type IDs that are allowed as new parent for this OrgUnit Type"],
+  "possible_group_sets": ["Array<Int> - List of possible GroupSet IDs for this OrgUnit Type"],
+  "editable_reference_forms": ["Array<Int> - List of reference Form ID that can be modified"]
+}
+```
+
+### Possible responses
+
+#### 201 - Created
+
+#### 400 - Bad request
+
+- A not nullable field was null or omitted
+- `org_unit_type_id` is not assigned to the given `project_id`
+- One or more `editable_fields` value is unknown
+- One or more `possible_parent_types` is not a suitable for as a parent for the given OrgUnit Type
+- One or more `editable_reference_forms` is not a reference form for the given OrgUnit Type
+
+#### 401 - Unauthorized
+
+- No authentication token or an invalid one was provided
+
+#### 403 - Forbidden
+
+- User doesn't have the proper permission
+
+#### 404 - Not found
+
+- `project_id` is not null and doesn't exist
+- `org_unit_type_id` is not null and doesn't exist
+- One or more `possible_parent_types` doesn't exist
+- One or more `possible_group_sets` doesn't exist
+- One or more `editable_reference_forms` doesn't exist
+
+
+## 2 - GET /api/change_requests_configs/
+
+API used to list all Change Request configurations.
+
+### Permissions
+
+- Change Request Configuration
+
+### Query parameters
+- page: Int (optional) - Current page (default: 1)
+- limit: Int (optional) - Number of entities returned by page (default: 20)
+- org_unit_type_id: Int (optional) - Id of the OrgUnitType to filter on
+- project_id: Int (optional) - Id of the Project to filter on
+
+### Possible responses
+
+#### 200 - OK
+
+```json
+{
+  "count": "Long",
+  "has_next": "Boolean",
+  "has_previous": "Boolean",
+  "page": "Long",
+  "pages": "Long",
+  "limit": "Long",
+  "results": [
+    {
+      "id": "Int - id in the database, I know Martin wants it",
+      "project_id": "Int - Project ID",
+      "project_name": "String - Project name",
```

Comment:
@bmonjoie do you need to have project data split into two fields at the same level, or do we do something like this:
```
"project": {
    "id": 3,
    "name": "uocgyxo project"
 },
 ``` 
 
 Same question for `org_unit_type`

Reply:
It's the same for me, I think I did it this way because this is how it is usually done in other endpoints.